server_tests/test/clientTests.js

```diff
@@ -1402,19 +1297,19 @@ describe("RUST API TEST SUITE", function() {
             }, timeSendAnother);
         });
 
-        it('Expect cameraInExclusiveUse Error. camera._bublTimelapse cannot be run when a video capture procedure is already active', function(done) {
+        it('Expect cameraInExclusiveUse Error. camera._bublTimelapse cannot be run when a video capture procedure is already active', function() {
             this.timeout(timeoutValue);
             var captureVideoDone = false;
             var startTime = Date.now();
             var timeSendAnother = 1000;
             var stopTime = 5000;
             var stopSent = false;
             var stopErr;
-            //instantiate the first camera._bublTimelapse command
-            testClient.bublCaptureVideo(sessionId, function(res) {
+            //instantiate the first camera._bublCaptureVideo command
+            return testClient.bublCaptureVideo(sessionId, function(res) {
```

Comment:
Same concerns here. Returning the promise here prevents the rest of the test from running, and the `setTimeout` logic is strange. Also spawning promises from within the status update of this call.